src/pages/Context.vue

```diff
@@ -4,7 +4,9 @@
 		<div v-if="!loading && context">
 			<div class="content first-row">
 				<div class="row">
-					<h1> {{ activeContext.iconName }}&nbsp; {{ activeContext.name }}</h1>
+					<h1 style="display: flex; align-items: center;">
+						<NcIconSvgWrapper :svg="icon" :size="40" style="display: inline-block;" />&nbsp; {{ activeContext.name }}
```

Comment:
Not sure why but that icon doesn't seem to render, in the navigation it works fine though.